src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -23,16 +35,53 @@ public HttpClientHandlerTest(ITestOutputHelper output)
         }
         
         [Fact]
-        public void SendAsync_SimpleGet_Success()
+        public async void SendAsync_SimpleGet_Success()
+        {
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+                    HttpResponseMessage response = await client.GetAsync(HttpTestServers.RemoteGetServer);
+                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    string responseContent = await response.Content.ReadAsStringAsync();
+                    _output.WriteLine(responseContent);
+                }
+            }
+        }
+
+        [Fact]
+        public async void SendAsync_SimpleHttpsGet_Success()
         {
-            var handler = new HttpClientHandler();
-            var client = new HttpClient(handler);
-            
-            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
-            var response = client.GetAsync(HttpTestServers.RemoteGetServer).Result;
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-            var responseContent = response.Content.ReadAsStringAsync().Result;
-            _output.WriteLine(responseContent);
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    HttpResponseMessage response = await client.GetAsync(HttpTestServers.SecureRemoteGetServer);
+                    string responseContent = await response.Content.ReadAsStringAsync();
+		                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    _output.WriteLine(responseContent);
```

Comment:
Our goal here is to validate that the HTTP request worked.  We set up the client/handler, we make the request, and then we need to validate the response, which might include validating more than one thing.  I don't think there's anything wrong with multiple asserts in a case like this: in fact, I think it's very desired.  Imagine if there were a method on Assert called ResponseIsCorrect, and the test looked like:
```C#
HttpResponseMessage response = await client.GetAsync(HttpTestServers.SecureRemoteGetServer);
string responseContent = await response.Content.ReadAsStringAsync();
Assert.ResponseIsCorrect(response, responseContent);
```
Presumably that verification method would check multiple things, as there are multiple things that could go wrong.  Obviously there isn't such an assert method, but it highlights that the intent of of "arrange-act-assert" is maintained.